client/src/utils/fight/FighterHolder.ts

```diff
@@ -1101,6 +1148,25 @@ export default class FighterHolder {
     });
   });
 
+  // Stop / restart actualizing zIndex
+  setAirborn = (airborn: boolean) => {
+    if (airborn) {
+      this.shadowSprite.alpha = 0;
+    } else {
+      this.shadowSprite.alpha = 1;
+      this.shadowSprite.y = 0;
+      this.container.zIndex = this.container.y;
+    }
+    this.#isAirborn = airborn;
+  };
+
+  // Display shadow on the right position (used while airborn)
+  updateShadow = () => {
+    const altitude = this.container.zIndex - this.container.y;
+    this.shadowSprite.y = altitude;
+    this.shadowSprite.alpha = (this.baseHeight - altitude * 0.9) / this.baseHeight;
+  };
```

Comment:
It would be better to also alter the shadow size, as it should be smaller when the fighter is higher up